### PR TITLE
Fix issue enable3d/enable3d#135

### PR DESCRIPTION
**Pull request by jairsnow**
Fix issue enable3d/enable3d#135

### CODE REVIEW

File: packages/common/src/extendedObject3D.ts [hunk 1]
```diff
@@ -77,11 +79,11 @@ export class ExtendedObject3D extends Object3D {
       /** Get AnimationAction by animation name. */
       get: (name: string) => this._animsGet(name),
       /**
-       * Play an animation.
-       * @param name Animation name.
-       * @param transitionDuration Transition duration in ms.
-       * @param loop Should the animation loop?
-       */
+      * Play an animation.
```
Comment (yandeu) [inline packages/common/src/extendedObject3D.ts:82]: Fix this by running `npm run format`.
(needs prettier)

File: packages/common/src/extendedObject3D.ts [hunk 2]
```diff
@@ -128,6 +130,32 @@ export class ExtendedObject3D extends Object3D {
...
     console.warn(`[enable3d] setAction(${name}) is deprecated. Use animation.play(${name}) instead!`)
   }
+
+  public traverse(callback:( object: ExtendedObject3D ) => any): void {
+    callback(this)
+    const children = this.children
+    for ( let i = 0, l = children.length; i < l; i ++ ) {
+      children[ i ].traverse( callback )
+    }
+  }
```
Comment (yandeu) [inline packages/common/src/extendedObject3D.ts:140]: I prefere:

```suggestion
public traverse(callback: (object: ExtendedObject3D) => any): void {
  super.traverse(callback as any)
}
```

File: packages/common/src/extendedObject3D.ts [hunk 3]
```diff
@@ -128,6 +130,32 @@ export class ExtendedObject3D extends Object3D {
...
+
+  public traverseVisible(callback:( object: ExtendedObject3D ) => any): void {
```
Comment (yandeu) [inline packages/common/src/extendedObject3D.ts:142]: Same as for `traverse`.

File: packages/common/src/extendedObject3D.ts [hunk 4]
```diff
@@ -128,6 +130,32 @@ export class ExtendedObject3D extends Object3D {
...
+    if ( this.visible === false ) return;
+    callback( this )
+    const children = this.children
+
+    for ( let i = 0, l = children.length; i < l; i ++ ) {
+      children[ i ].traverseVisible( callback )
+    }
+  }
+  
+  public traverseAncestors(callback:( object: ExtendedObject3D ) => any): void {
```
Comment (yandeu) [inline packages/common/src/extendedObject3D.ts:152]: Same as for `traverse`.

File: packages/common/src/extendedObject3D.ts (hunk 2 above)
Comment (jairsnow) [reply packages/common/src/extendedObject3D.ts]: Yeah, in this way is 1000 time better! Thanks!

**Review by yandeu — APPROVED** [review]
Thx

**Pull request merged**